WORKSPACE

```diff
@@ -17,7 +17,7 @@ workspace(name = "build_bazel_rules_typescript")
 git_repository(
     name = "build_bazel_rules_nodejs",
     remote = "https://github.com/bazelbuild/rules_nodejs",
-    tag = "0.2.2",
+    commit = "1a3a0de",
```

Comment:
Please use a full SHA1

1a3a0dea51c0e53a2871fbfaba3be178a308a7a3

collisions in the reduced SHA1 address space are almost guaranteed as the commit count grows. Thanks :birthday: paradox!

Reply:
don't need to include this in the PR, sorry for so much noise here. Still trying to figure out how to do CI